app/routes/speak.tsx

```diff
@@ -35,7 +35,11 @@ export default function About() {
 			<UnorderedList>
 				<li>
 					Present a dry run of your talk the week before, so we can check it for
-					accessibility issues and make sure it runs the expected duration
+					accessibility issues and make sure it runs to the expected duration:
+					<ul>
+						<li>Main talk: ~20 minutes</li>
+						<li>Lightning talk: ~10 minutes</li>
```

Comment:
Could we use `about` instead of `~`? It would be more accessible as screen readers (tested with VoiceOver on macOS) says "tilde 20 minutes":

```suggestion
						<li>Main talk: about 20 minutes</li>
						<li>Lightning talk: about 10 minutes</li>
```